app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The post should belong to an `Author` not a user, kindly fix this.
e.g
`belongs_to :author, class_name: 'User'`